README.md

```diff
@@ -44,6 +48,20 @@ Read the [installation guide](/docs/setup/install.md) to learn more.
 - ✔️ Security: RBAC based accessibility.
 - ✔️ Stable release cycle(2-3 months) for new features, bugfixes, cleanups.
 
+## Installation
+
+**Requires Kubernetes 1.26 or newer**.
```

Comment:
Can we say that we follow Kubernetes release process?

In 1-2 months I think we would want to bump this to Kubernetes 1.27..

Reply:
Sure, so something like:

**Maintains support for latest 3 Kubernetes minor versions. Current: 1.27, 1.28, 1.29**

(I know we currently run e2e-tests with 1.26 as well, but we should remove this and just focus on support for latest 3 minors, to align with upstream k8s).

What are your thoughts on this?

Reply:
My goal would be to avoid having to PR to keep these versions up to date..

Maintains support for latest 3 Kubernetes minor versions.

Reply:
Ok I added a line to "production readiness" bullets about this, and then here (installation instructiosn) I mentioned one of the last 3 minor versions is required.